benches/galen.rs

```diff
@@ -252,8 +266,10 @@ fn main() -> Result<()> {
                     },
                 )
                 .unwrap();
-            outp.inspect(|zs: &OrdZSet<_, _>| println!("outp: {}", zs.len()));
-            outq.inspect(|zs: &OrdZSet<_, _>| println!("outq: {}", zs.len()));
+            outp.gather(0)
```

Comment:
Not necessarily.  A multithreaded consumer (e.g., another DBSP circuit) might prefer to get data from many worker threads in parallel instead of a gathered input.  In DD you typically invoke user callback from a final `inspect` operator attached to each worker.